layers/+chat/circe/packages.el

```diff
@@ -0,0 +1,151 @@
+;;; packages.el --- Circe layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Chris Barrett <chris.d.barrett@me.com>
+;; Author: Aidan Nyquist <contact@aidannyquist.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Code:
+
+(defconst circe-packages
+  '(circe
+    circe-notifications
+    persp-mode
+    helm
+    helm-circe
```

Comment:
all packages you use need to be in this list. You're still missing `circe-color-nicks`, `lui`, and `lui-autopaste`. if some of them rely on other packages to be loaded first you can use something like     `(<package> :toggle (configuration-layer/package-usedp '<required-package>))`